iris_grib/_load_convert.py

```diff
@@ -128,6 +131,21 @@
 # NOTE: Our test data contains the value 2, which is all we currently support.
 # The exact interpretation of this is still unclear.
 
+# See Code Table 4.15 for full spatial processing type descriptors:
+# http://apps.ecmwf.int/codes/grib/format/grib2/ctables/4/15
+
+# InterpolationParameters(statistical process (octet 35), number of points
+# used in interpolation (octet 37))
+_SPATIAL_PROCESSING_TYPES = {
+    0: InterpolationParameters('cell_method', 0),  # No interpolation
+    1: InterpolationParameters(None, 4),  # Bilinear interpolation
+    2: InterpolationParameters(None, 4),  # Bicubic interpolation
+    3: InterpolationParameters(None, 1),  # Nearest neighbour interpolation
```

Comment:
@kaedonkers Was there a reason for the statistical process code of nearest neighbour interpolation being 0?  I changed it to None because I thought that made more sense, but maybe I am missing something?